src/AppInstallerCommonCore/Public/winget/ExperimentalFeature.h

```diff
@@ -23,6 +23,7 @@ namespace AppInstaller::Settings
             Dependencies = 0x1,
             // Before making DirectMSI non-experimental, it should be part of manifest validation.
             DirectMSI = 0x2,
+            PortableInstall = 0x3,
```

Comment:
These are bit flags values.  Most of the code doesn't care, but `ExperimentalFeature::GetAllFeatures` does.  And being able to support querying for multiple flags at the same time might in a future where we want that (why it was done that way in the first place.

Reply:
Ah my mistake, I didn't realize these were bit flags, changed the flag values for `PortableInstall` and `Max` so that the experimental feature shows up for `ExperimentalFeature::GetAllFeatures`